src/Nordea/Components/DropdownFilter.elm

```diff
@@ -134,17 +133,43 @@ view attrs (DropdownFilter config) =
                 |> String.contains (String.toLower config.input)
 
         searchMatches =
+            let
+                viewHeader text =
+                    Html.li
+                        [ css
+                            [ color Colors.gray
+                            , margin2 (rem 0) (rem 0.625)
+                            , fontSize (rem 0.75)
+                            , fontWeight (int 400)
+                            , lineHeight (rem 1)
+                            ]
```

Comment:
Do this need to be a custom styled text, or can we use of the the text components? (and maybe add a few overrides). May be worth discussing the header style with Thea, Andreas has already complained that they are somewhat hard to read with the current style

Reply:
Used the `TextTinyLight` variant for now, seems like the closest to what we want😊 I'll have a talk with Thea a day she is at the office